### DISCUSSION TITLE
Happy to help if there is a brief intro about this idea

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [korandoru/town#16] [General]

**Comment by tisonkun** [comment]
cc @LinuxSuRen 

首先，欢迎任何形式的开源相关的博客投递到[【集体智慧】](https://town.korandoru.io/blog)页面。我会做一个编辑审核。

其次，我把开源社群当中的角色和关注角度分成了：

1. 个人参与。
2. 社群建设。
3. 企业沙盘。
4. 数字看板。

## 个人参与

从一个独立开发者的角度出发，解决认识开源和如何参与开源的问题。例如，

* 如何拒绝 Maintainer 不合理的请求？
* 如何发现适合参与的项目？
* 如何在开源社群当中得心应手？
* ...

## 社群建设

从社群的维护者、组织者的角度出发，解决如何建设和运营开源社群的问题。例如，

* [Apache 衡量项目成熟度的方式](https://town.korandoru.io/communities/apache-maturity-model)
* Committer 的标准
* Review Contribution 的标准
* ...

## 企业沙盘

企业尤其是软件技术企业，现在已经不可能脱离开源软件运行自己的业务了。从开源软件的使用者、开发跟随者到领导者，如何认识企业运营与开源社群发展之间的关系。最终根据自己的状况，制定开源策略，在企业沙盘上推演。例如，

* [诱导转向的伪开源战略](https://town.korandoru.io/enterprise/bait-and-switch-fauxpen-source-strategy)
* 开源软件不赚钱：从软件、产品到商品
* ...

## 数字看板

一开始，我想做一个数字化的社群指标方案。但是看起来这个不太适合跟这个项目耦合在一起，而且做起来要泛用还是很麻烦的。

相反，未来的发展方向应该是把大家已经用上的指标方案进行描述和展示，尽可能提供复现方式。比如当前放在那的 Pulsar 每月最活跃参与者，说清楚是怎么做的，方便其他访客借鉴，落实到自己的工作当中。
